Quickstore/pages-styles/signin.css

```diff
@@ -0,0 +1,50 @@
+@import url('/Quickstore/pages-styles/landingpage.css');
+@import url('/Quickstore/pages-styles/color.css');
+.login-area{
+    height: 90vh;
+    width: 100%;
+    background-color:  var(--loginbackground-color); 
+    display: flex; 
+    justify-content: center;
+    align-items: center; 
+}
+.login-container{
+    height: 25rem;
+    width: 22rem;
+    background-color: var(--default-color);
+    margin: 5rem;
+    text-align: center;
+    
+}
+ 
+.login-container h2{
+    text-align: center;
+    margin: 2rem 0 2rem;
+}
+.input-box{
+    height: 2rem;
+    width: 15rem;
+    margin-bottom: 2rem;
+}
+.search-input{
+    height: 2rem;
+    width: 17rem;
+}
+.login-link { 
+    height: 2rem;
+    width: 15rem;
+    margin: 1rem 3.6rem;
+}
+
+.forget-password{
+    margin-left: 2rem;
+    text-decoration: none;
+}
+.content{
```

Comment:
If you are using this class somewhere have a descriptive name or just remove it.